product_pricelist_revision/models/pricelist.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Carlos Dauden - Tecnativa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+    _rec_names_search = ["name"]
```

Comment:
Don't modify this, just do the extend as you did.

Reply:
If it is not initialized here, it does not allow extending or making direct assignment